docs/developers/debugging/tf_integrations_test_repro.md

```diff
@@ -0,0 +1,37 @@
+# Debugging failures in TF/TFLite integration tests.
+
+These are steps to reproduce/address failures in TF/TFLite integration tests. All steps here
+assume starting from the IREE root directory.
+
+1. First setup the python environment as described [here](https://google.github.io/iree/building-from-source/python-bindings-and-importers/#environment-setup).
+
+```
+python -m iree.venv
+source iree.venv/bin/activate
+```
+
+2. Install latest IREE release binaries. The importers are not expected to change much, so using the release binaries should work for most cases
+
+```
+python -m pip install iree-compiler iree-runtime iree-tools-tf iree-tools-tflite -f https://github.com/google/iree/releases/latest
+```
+
+3. Install TF nightly
+
+```
+python -m pip install tf-nightly Pillow
```

Comment:
is Pillow required? I'd expect to also see it in https://github.com/google/iree/blob/main/bindings/python/build_requirements.txt or a similar file if so

Reply:
Seems like TF-nightly needs pillow. I dont know if we need it by default (you need to probably check with Stella)